packages/viz/__tests__/viewport-features/cases/lines/lines.ts

```diff
@@ -0,0 +1,51 @@
+import { Vector3 } from '@math.gl/core';
+import { ViewportFrustumPlanes } from '../../../../src/lib/interactivity/viewport-features/geometry/types';
+import { ViewportFeaturesOptions } from '../../../../src/lib/interactivity/viewport-features/ViewportFeaturesGenerator';
+import * as tiles from './tiles.json';
+
+export {
+  name,
+  tiles,
+  viewportFeaturesOptions,
+  viewportFeaturesResult,
+  frustumPlanes
+};
+
+const name = 'Lines';
+
+const viewportFeaturesOptions: ViewportFeaturesOptions = {
+  properties: ['street'],
+  aggregations: {}
+};
+
+const viewportFeaturesResult = {
+  features: [{ street: 'FITZGERALD' }],
+  aggregations: {}
+};
+
+const frustumPlanes: ViewportFrustumPlanes = {
+  near: {
+    distance: 0.0023256811732884296,
```

Comment:
Nice approach with all those cases in a folder ✨! 

How did you generate these use cases? Did you maybe use a small .html setup, with a known width and height, etc., to observe and validate from real values?  I'm thinking about revisiting this in the future, trying to get some context on how to modify or check a particular spec (and they don't look like obvious to me...) 

Could add some brief notes on them to help a bit?

Reply:
Also, I think that we could probably split the tests in 2 parts: detecting the features in viewport (dealing with these complex geometry computations) and calculating aggregations (much simpler and easier to test, using just a numeric field)

Reply:
I got the viewport frustum planes and the rest of the data inspecting and debugging the code with the browser. I'd say that is very unlikely that we need to change them, but nevertheless I would put some instructions just in case.

Given that I am currently splitting both features, viewport features and aggregations, those tests will be split as well. So no problem.